pkg/segment/structs/segstructs.go

```diff
@@ -461,6 +461,14 @@ func (ma *MeasureAggregator) String() string {
 	return ma.StrEnc
 }
 
+func GetMeasureAggregatorStrEncColumns(measureAggs []*MeasureAggregator) []string {
+	var columns []string
+	for _, ma := range measureAggs {
+		columns = append(columns, ma.String())
+	}
+	return columns
+}
```

Comment:
No need to change this. I just want to point out that this could be a good type of function to make generic: one that takes a list of some type and a function on that type to output another type. Maybe some other code could also use a similar function

```
func convertSlice[T, R](slice []T, func(T) R) []R {....}
```

Reply:
yes got it.